script/release/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM golang:1.17.7 AS build
+
+ARG CONTAINERD_VERSION=1.5.7
+ARG NYDUS_BUILER_VERSION=v2.0.0-rc.6
+
+# Install acceld
+ADD ./ /accel
+RUN make -C /accel install-check-tools
+RUN make -C /accel
+
+# Install containerd
+RUN wget https://github.com/containerd/containerd/releases/download/v$CONTAINERD_VERSION/containerd-$CONTAINERD_VERSION-linux-amd64.tar.gz
+RUN wget https://github.com/containerd/containerd/releases/download/v$CONTAINERD_VERSION/containerd-$CONTAINERD_VERSION-linux-amd64.tar.gz.sha256sum
```

Comment:
checksum should be hardcoded to Dockerfile or some other files.

Reply:
What is the purpose of sha256sum being hardcoded to Dockerfile, is there a reference example?

Reply:
For checking that the downloaded binary is exactly same as the expected one in the Dockerfile.

kind example: https://github.com/kubernetes-sigs/kind/blob/main/images/base/Dockerfile#L30-L36
nerdctl example: https://github.com/containerd/nerdctl/tree/master/Dockerfile.d/SHA256SUMS.d

Reply:
Thanks, fixed.